src/main/java/com/gluonhq/substrate/util/ProcessRunner.java

```diff
@@ -288,7 +288,7 @@ private Process setupProcess(String processName, File directory) throws IOExcept
         return pb.start();
     }
 
-    private static Thread mergeProcessOutput(final InputStream is, final StringBuffer sb) {
+    private static Thread mergeProcessOutput(final InputStream is, final StringBuffer sb, final boolean info) {
```

Comment:
What is the use for passing the `sb` and `info` fields to this method? The method is always invoked with `ProcessRunner.answer` and `ProcessRunner.info`, so you might as well use those fields directly?

Reply:
Not sure if I follow? `mergeProcessOutput` is static, but those files are not, so I can't use them?

Reply:
Good point, why don't we remove static from the method as well as it is called as if it was an instance method.

Reply:
Ok, makes sense. Pushed